### ISSUE TITLE
BBC External link promotion (Disclaimer Work)

### ISSUE THREAD

**Issue by DarioR01** [bbc/simorgh#9996]
**Is your feature request related to a problem? Please describe.**
After the happenings that involved the [Disclaimer component](https://github.com/bbc/simorgh/tree/latest/src/app/containers/Disclaimer), it is not currently ready to be shipped or used.
The current usage that we are making of this Disclaimer component is to promote BBC external links. However, the original idea behind the Disclaimer was to use this component as a normal disclaimer.

After a confused and rushed implementation, we are currently using this component as a tool for sharing BBC external links on the Russian service. However, we are using the accessibility requirements of the original disclaimer.

The Russian editorial team is seeing a substantial increase in telegram subscriptions and might consider promoting other BBC's social media in the future.
It looks that the component will be used as a promotional component by the Russian service, and we should treat it accordingly.

**Describe the solution you'd like**
### As a Promotion of BBC links
Follow these [UX A11y](https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/622ef23984e71f48dcf4f80e) designs.

Further steps to take:
- [x] Make sure that the text uses lower case and is not all written in caps.
- [x] We are currently announcing `disclaimer` for this component. We should be announcing `information`: Russian translation: `информация`. Use `information` as a fallback for all the other services.
- [x] Once you apply the correct styling (lower case), review with UX to make sure the changes do not need further styling changes (e.g. Will the line-height be the same? Are we using the same font-size? etc...) 

**Testing notes**
[Tester to complete]

**Dev Insight:**

- Consider changing the name of the component to `information` or anything that reflects the use we are making of this.

- [x] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**
I have previously created this [PR](https://github.com/bbc/simorgh/pull/9995) that converts the disclaimer component into a tool for sharing BBC external links. That PR is missing the following:
- Translations for the new Region's label.
- The text is still in caps and should be converted to normal lower case text.
- Consider changing the component's name to something else as this is not a Disclaimer!!!

**Old Disclaimer a11y requirements:** [A11y disclaimer a/c](https://paper.dropbox.com/doc/kR9WZaIuJi73YyUIl3XKw?cl=en_US).
